aiida/cmdline/params/types/profile.py

```diff
@@ -16,7 +16,12 @@
 
 
 class ProfileParamType(LabelStringType):
-    """The profile parameter type for click."""
+    """The profile parameter type for click.
+
+    This parameter type requires the command that uses it to define the ``context_class`` class attribute to be the
+    :class:`aiida.cmdline.groups.verdi.VerdiContext` class, as that is responsible for creating the user defined object
+    ``obj`` on the context and loads the instance config.
```

Comment:
Is there a way to validate this?
If this is straightforward (even if just indirect by checking the `ctx.obj`) a check + exception might be a more effective way of enforcing this than via documentation.

Reply:
Can do, will add it.